packages/kununu-icons/CheckSuccessOutline.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
app-reviews was using it as CheckSuccess, I'll have to open a PR to change it and import CheckSuccessOutline there instead.